packages/cozy-client/src/store/queries.js

```diff
@@ -7,18 +7,26 @@ import uniq from 'lodash/uniq'
 import orderBy from 'lodash/orderBy'
 import isArray from 'lodash/isArray'
 import isString from 'lodash/isString'
+import get from 'lodash/get'
+import sift from 'sift'
+
+import flag from 'cozy-flags'
 
 import { getDocumentFromSlice } from './documents'
 import { isReceivingMutationResult } from './mutations'
 import { properId } from './helpers'
-import sift from 'sift'
-import get from 'lodash/get'
 
 const INIT_QUERY = 'INIT_QUERY'
 const LOAD_QUERY = 'LOAD_QUERY'
 const RECEIVE_QUERY_RESULT = 'RECEIVE_QUERY_RESULT'
 const RECEIVE_QUERY_ERROR = 'RECEIVE_QUERY_ERROR'
 
+// Read if the devtools are open to store the execution time
+// This is done at runtime to not read the value everytime
+// we receive a result. So you have to refresh your page
+// in order to get the stats
+const executionStatsEnabled = flag('perfs.execution_stats')
```

Comment:
This is really not a problem to read the value everytime you receive a result ;) Flags are stored in memory so that we do not have to check localStorage for a read.

Reply:
I don't like to put a function call in a method we use a lot. I know that modern browsers have a lot of optimizations for that specific use case but still think that if we can avoid it we should 🤷‍♂️

Reply:
You do not like hooks ? :D